packages/storage/jest.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from '@fuels/test-lib/config';
```

Comment:
Change the name of the package to `react-storage`

Reply:
this isn't a react specific project, can be used anywhere